Marlin/src/MarlinCore.cpp

```diff
@@ -920,6 +924,14 @@ void setup() {
     SERIAL_ECHO_MSG("start");
   #endif
 
+  #if BOTH(HAS_TFT_LVGL_UI, USE_WIFI_FUNCTION)
+    #define SERIAL_PORT_2 -1
+    mks_esp_wifi_init();
+    WIFISERIAL.begin(WIFI_BAUDRATE);
+    serial_connect_timeout = millis() + 1000UL;
+    while (/*!WIFISERIAL && */PENDING(millis(), serial_connect_timeout)) { /*nada*/ }
```

Comment:
This is going to add a full second of dead time to the startup. Are you sure you don't want to examine `WIFISERIAL` for a non-zero value that could indicate a very quick connection success?

Reply:
I think we can talk about wifi serial later, as it is hardcoded disabled by now. And there's a lot of thinks to review about MKS wifi module....